ansible/hosts.example

```diff
@@ -59,6 +59,12 @@ all:
       #   lti11_enabled and lti13_enabled are mutually exclusive, therefore
       #   set one to true to enable the desired lti auth standard.
 
+      ### Genaral Options
+      #-------------------
+      # provide a list of admin users separated by commas. Usernames should represent
+      # the normalized username with logging into the system.
+      # admin_users: foo,biz,bar
```

Comment:
should it be plural ? in `ansible/roles/jupyterhub/templates/env.jhub.j2` file is used as singular

Reply:
It should be single, yes. I'll provide an update.